src/StatusQ/Controls/StatusSlider.qml

```diff
@@ -31,7 +31,7 @@ Slider {
         id: bgRect
 
         x: root.leftPadding
-        y: root.topPadding
+        y: root.topPadding + root.availableHeight / 2 - height / 2
```

Comment:
could you add parenthesis just to be sure the this is interpreted as we want? I've seen cases where the calculations were weird because of "free will" interpretation, for instance it was calculating (root.availableHeight / (2 - height / 2) and so on...

Reply:
Wow :O - never heart of "free will" interpretation - sounds crazy and I dont remember to meet it even I had pretty complex constructions. But ok I will write it as `topPaddin + (available - height) / 2` , hope in case of "free will" there are no other interpretations.

Reply:
Well it definitely helps readability and that's a good thing. It doesn't add any overhead